dao/accumulo.rya/src/main/java/org/apache/rya/accumulo/AccumuloRdfConfiguration.java

```diff
@@ -68,10 +48,88 @@ public AccumuloRdfConfiguration(Configuration other) {
         super(other);
     }
 
+    public AccumuloRdfConfigurationBuilder getBuilder() {
+    	return new AccumuloRdfConfigurationBuilder();
+    }
+    
+	/**
+	 * Creates an AccumuloRdfConfiguration object from a Properties file.  This method assumes
+	 * that all values in the Properties file are Strings and that the Properties file uses the keys below.
+	 * See accumulo/rya/src/test/resources/properties/rya.properties for an example.
+	 * <br>
+	 * <ul>
+	 * <li>"accumulo.auths" - String of Accumulo authorizations. Default is empty String.
+	 * <li>"accumulo.visibilities" - String of Accumulo visibilities assigned to ingested triples.
+	 * <li>"accumulo.instance" - Accumulo instance name (required)
+	 * <li>"accumulo.user" - Accumulo user (required)
+	 * <li>"accumulo.password" - Accumulo password (required)
+	 * <li>"accumulo.rya.prefix" - Prefix for Accumulo backed Rya instance.  Default is "rya_"
+     * <li>"accumulo.zookeepers" - Zookeepers for underlying Accumulo instance (required if not using Mock)
+     * <li>"use.mock" - Use a MockAccumulo instance as back-end for Rya instance.  Default is false.
+     * <li>"use.prefix.hashing" - Use prefix hashing for triples.  Helps avoid hot-spotting.  Default is false.
+     * <li>"use.count.stats" - Use triple pattern cardinalities for query optimization.   Default is false.
+     * <li>"use.join.selectivity" - Use join selectivity for query optimization.  Default is false.
+     * <li>"use.display.plan" - Display query plan during evaluation.  Useful for debugging.   Default is true.
+     * <li>"use.inference" - Use backward chaining inference during query evaluation.   Default is false.
+     * </ul>
+	 * <br>
+	 * @param props - Properties file containing Accumulo specific configuration parameters
+	 * @return AccumumuloRdfConfiguration with properties set
+	 */
+    
+    public static AccumuloRdfConfiguration fromProperties(Properties props) {
+    	return AccumuloRdfConfigurationBuilder.fromProperties(props).build();
+    }
+    
     @Override
     public AccumuloRdfConfiguration clone() {
         return new AccumuloRdfConfiguration(this);
     }
+    
+    public void setAccumuloUser(String user) {
+    	Preconditions.checkNotNull(user);
```

Comment:
Maybe use Objects.requireNonNull since this is built into Java 8 instead of using Guava.